backend/sublime/events_manual.go

```diff
@@ -172,7 +172,7 @@ func (c *OnQueryContextGlue) onQueryContext(v *backend.View, key string, operato
 	defer ret.Decref()
 
 	//	if ret != nil {
-	log.Fine("onQueryContext: %v, %v", pv, ret.Base())
+	// log.Fine("onQueryContext: %v, %v", pv, ret.Base())
```

Comment:
It seemed extraneous and was cluttering up the log output. Happy to add it back in if we feel strongly about it.